core-models/src/main/scala/com/mesosphere/usi/core/models/RunSpec.scala

```diff
@@ -16,4 +17,11 @@ import com.mesosphere.usi.core.models.resources.ResourceRequirement
 case class RunSpec(
     resourceRequirements: Seq[ResourceRequirement],
     shellCommand: String,
-    fetch: Seq[FetchUri] = Seq.empty)
+    fetch: Seq[FetchUri] = Seq.empty) {
+
+  def this(
```

Comment:
I'll repost my comment on slack:

shouldn't we do all the java APIs in a separate namespace and ideally write them in java? Then we know they are java friendly :slightly_smiling_face: Akka has two namespaces as well and the ones for java are written in ... wait for it... Java :slightly_smiling_face: